src/platforms/common/stm32/traceswoasync.c

```diff
@@ -29,6 +29,8 @@
 /* TDO/TRACESWO signal comes into the SWOUSART RX pin. */
 
 #include <stdatomic.h>
+#include <malloc.h>
+#include <errno.h>
```

Comment:
These in particular should be included after `general.h` so bad things don't happen with the feature macros. Ideally the `stdatomic.h` include should really also be moved after.

Reply:
Could move.